manifests/z/Zoom/Zoom/5.11.10.8200/Zoom.Zoom.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Zoom.Zoom
+PackageVersion: 5.11.10.8200

```

```suggestion
PackageVersion: 5.11.8200
```
This must match Control Panel.